lib/urldata.h

```diff
@@ -511,6 +509,28 @@ enum upgrade101 {
   UPGR101_WORKING             /* talking upgraded protocol */
 };
 
+struct dohresponse {
+  unsigned char *memory;
+  size_t size;
+};
+
+/* one of these for each DOH HTTP request */
```

Comment:
Nitpick, but should this say "DOH HTTPS request"?

Reply:
correct!

Reply:
or perhaps just "each DoH request" ...